codebuild/bin/s2n_codebuild.sh

```diff
@@ -65,18 +65,26 @@ if [[ -n "$S2N_NO_PQ" ]]; then
     CMAKE_PQ_OPTION="S2N_NO_PQ=True"
 fi
 
-source codebuild/bin/s2n_apache2.sh
-APACHE_CERT_DIR="$(pwd)/tests/pems"
+setup_integration_v2_tests() {
+    # Start the apache server if the list of tests isn't defined, meaning all tests
```

Comment:
I'd be explicit in what this does: setup_apache_server

Reply:
Fixed!